### PR TITLE
Updates readme, separates previous content

### PR DESCRIPTION
**Pull request by jorge0136**
Does anyone have the blog post that we used to build the GRU? Let me know what you think!

I think we should organize our different experiments into folders, to make it easier to see our attempts over time.

### CODE REVIEW

**Comment by briannabrownOE** [comment]
> Does anyone have the blog post that we used to build the GRU? Let me know what you think!
> 
> I think we should organize our different experiments into folders, to make it easier to see our attempts over time.

**Comment by briannabrownOE** [comment]
> Does anyone have the blog post that we used to build the GRU? Let me know what you think!

I think this is what we used: https://www.kaggle.com/shivamb/beginners-guide-to-text-generation-using-lstms

File: README.md [hunk 1]
```diff
@@ -1,95 +1,26 @@
...
 
-## App
+### Progress Thus Far
 
-* Render model results as plain text on an html page
-* Build flask prototype
-* Get a domain
-* Website-to-model 
-* Feedback from within UI
+* Attempt One: GRU with corpus embedding. Largely based on this blog post. TODO: Add blog post link.
```
Comment (briannabrownOE) [inline README.md]: here ya go! https://www.kaggle.com/shivamb/beginners-guide-to-text-generation-using-lstms

File: README.md [hunk 2]
```diff
@@ -1,95 +1,26 @@
...
-
-Things we've done:
-- Tested the tensorflow character-level, turns out not work well.
-- Added word embedding (glove) to the fortune cookie model
-
-To-do:
-- finish the word embedding
-
+1. It has been difficult to find the volume of data required to train a natrual language processing model on fortune cookies. There are not very many fortune cookie databases out in the wild.
+2. Fortune cookies are built from somewhat atypical English, which has made the lack of data more difficult.
```
Comment (briannabrownOE) [inline README.md]: I'd also maybe add to this point or one under it something about the difficulty that comes with testing. Challenges around verifying how "fortune cookie-y" our generated fortunes are.

Comment (ddong63) [reply README.md]: Agreed! I remembered we tried to score it manually, but still having the questions on finding an objective metric.

File: log.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+Move ipython notebook to a .py file, cleaned up the script.
+
+To-do:
+- make a python package
+- add flask
+
+### Nov 7th & Oct 30th
+
+Added word embedding as an additional layer.
+
```
Comment (ddong63) [inline log.md]: The dates were disordered somehow. Would you mind ordering them?

**Pull request merged**